src/sentry/services/hybrid_cloud/__init__.py

```diff
@@ -154,8 +163,15 @@ def close(self):
         pass
 
     def get_organizations(
-        self, user_id: Optional[id], scope: Optional[str], only_visible: bool
+        self, user_id: int, scope: Optional[str], only_visible: bool
     ) -> List[ApiOrganization]:
+        organizations = self._query_organizations(user_id, scope, only_visible)
+        membership = OrganizationMember.objects.filter(user_id=user_id)
+        return [self._serialize_organization(o, membership) for o in organizations]
+
+    def _query_organizations(
+        self, user_id: int, scope: Optional[str], only_visible: bool
+    ) -> List[Organization]:
```

Comment:
So this logic is a faithful copy from `Organization.objects.get_for_user`.  I really, wish I could call that function directly as the underlying implementation but I cannot due to the context it assumes (full organization relationships).  I didn't want to stuff my service call into that method, I felt it was wiser and narrower change scope to simply copy its implementation here and use it in a few specific call sites.  That said, I left a comment in that function to mind people to look for any relevant changes they may make there, and put them here.

It's a calculated risk.  I felt that the risk of directly modifying that highly utilized function right now was high, and the risk that the same function would receive significant changes in the near future lower.